src/test/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEvalIntegrationTest.java

```diff
@@ -106,6 +106,42 @@ public void testFundamentalsCountVariantsSNPsAndIndelsWithNovelty() throws IOExc
         spec.executeTest(name, this);
     }
 
+    @Test
+    public void testContigStratWithUserSuppliedIntervals() throws IOException {
+        String name = "testContigStratWithUserSuppliedIntervals";
+
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                " -R " + b37Reference +
+                        " --eval " + snpEffVcf +
+                        " -no-ev" +
+                        " -EV CountVariants" +
+                        " -no-st" +
+                        " -ST Contig" +
+                        " -L 1:1-1480226" +
+                        " -O %s",
+                Arrays.asList(getExpectedFile(name))
+        );
+        spec.executeTest(name, this);
+    }
+
+    @Test
+    public void testContigStratWithUserSuppliedIntervals2() throws IOException {
+        String name = "testContigStratWithUserSuppliedIntervals2";
+
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                " -R " + b37Reference +
+                        " --eval " + snpEffVcf +
+                        " -no-ev" +
+                        " -EV CountVariants" +
+                        " -no-st" +
+                        " -ST Contig" +
+                        " -L 1" +
+                        " -O %s",
+                Arrays.asList(getExpectedFile(name))
+        );
+        spec.executeTest(name, this);
+    }
+
```

Comment:
These two tests are identical except for the file name and interval, so they can be replaced with a single test that uses a `@DataProvider`. That will also nicely document the test files contents. Also please add a test case with no intervals specified.

Reply:
Not sure if you've used data providers before - if not let me know and I'll add one here.

Reply:
yes, i've used them. i reworked the test so it doesnt need the static test files anymore. i dont know what GATK's vision is in terms of having the full output checked in vs. testing specific features of the output. most of VariantEvalIntegration test uses the former since it was ported from GATK3, but i assume you dont want infinite test outputs checked in.

Reply:
This seems much more convoluted and error-prone than the previous version, which was much easier to comprehend.